src/containers/PerfherderGraphContainer/index.jsx

```diff
@@ -87,6 +98,7 @@ class PerfherderGraphContainer extends Component {
             tooltipModel={tooltipModel}
             series={options.series}
             canvas={canvas}
+            isLocked={tooltipIsLocked}
```

Comment:
I'm sending `isLocked` as props to the tooltip in order to throw a customized message to the user.